libraries/AP_Baro/AP_Baro_MS5611.cpp

```diff
@@ -342,6 +342,7 @@ void AP_Baro_MS56XX::update()
     }
 }
 
+static bool first_calc{true};
```

Comment:
This would never be allowed here, but in any case, why do you even need a static?